### PR TITLE
Hotfix/table toolbar

### PR DESCRIPTION
**Pull request by ohmyguigs**
# What does this PR do?
 - align table toolbar icons vertically
 - removes table toolbar button labels in favor of hints (soon to be tooltips)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,11 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Fixed
+
+- **Table** toolbar buttons alignment.
+- **Table** use hints in favor of button labels in toolbar buttons.
```
Comment (kevinch) [inline CHANGELOG.md]:
```suggestion
- **Table** uses hints in favor of button labels in toolbar buttons.
```

File: react/components/Table/README.md [hunk 2]
```diff
@@ -382,3 +377,81 @@ class ResourceListExample extends React.Component {
...
+  itemsLength: sampleData.items.length,
+  emptyStateLabel: 'Nothing to show.',
+}
+
+class ResourceListExample extends React.Component {
+  constructor() {
+    super()
+
+    this.state = initialState
+
```
Comment (vitoriaHeliane) [inline react/components/Table/README.md]: Remove theses white lines.

File: react/components/Table/Toolbar.js [hunk 3]
```diff
@@ -1,20 +1,20 @@
...
 import Button from '../Button'
 import ButtonWithIcon from '../ButtonWithIcon'
 import Toggle from '../Toggle'
-import IconCaretDown from '../icon/CaretDown'
 import IconColumns from '../icon/Columns'
 import IconDensity from '../icon/Density'
 import IconDownload from '../icon/Download'
 import IconPlus from '../icon/Plus'
 import IconUpload from '../icon/Upload'
+import IconOptionsDots from '../icon/OptionsDots'
```
Comment (ericreis) [inline react/components/Table/Toolbar.js]:
```suggestion
import IconOptionsDots from '../icon/OptionsDots'

```

File: react/components/Table/Toolbar.js [hunk 4]
```diff
@@ -114,14 +105,10 @@ class Toolbar extends PureComponent {
...
     } = this.props
-    const {
-      isFieldsBoxVisible,
-      isExtraActionsBoxVisible,
-      isDensityBoxVisible,
-    } = this.state
-    const isDownloadVisible = download && download.label
-    const isUploadVisible = upload && upload.label
-    const isFieldsVisible = fields && fields.label
+    const { isFieldsBoxVisible, isDensityBoxVisible } = this.state
```
Comment (ericreis) [inline react/components/Table/Toolbar.js]:
```suggestion
    
    const { isFieldsBoxVisible, isDensityBoxVisible } = this.state
```

**Pull request merged**